l2-contracts/contracts/bridge/L2AssetRouter.sol

```diff
@@ -22,13 +21,16 @@ import {EmptyAddress, InvalidCaller} from "../L2ContractErrors.sol";
 /// @custom:security-contact security@matterlabs.dev
 /// @notice The "default" bridge implementation for the ERC20 tokens. Note, that it does not
 /// support any custom token logic, i.e. rebase tokens' functionality is not supported.
-contract L2AssetRouter is IL2AssetRouter, ILegacyL2SharedBridge, Initializable {
+contract L2AssetRouter is IL2AssetRouter, Initializable {
     /// @dev Chain ID of Era for legacy reasons
     uint256 public immutable ERA_CHAIN_ID;
 
     /// @dev Chain ID of L1 for bridging reasons
     uint256 public immutable L1_CHAIN_ID;
 
+    /// @dev The address of the L2 legacy shared bridge.
+    address public L2_LEGACY_SHARED_BRIDGE;
+
```

Comment:
not from this line, but could you please remove the `DEPRECATED` fields from the L2AssetRouter if is not the shared bridge anymore?

Reply:
also `l1TokenAddress` is not needed there